### ISSUE TITLE
AttributeError on Django 1.10

### ISSUE THREAD

**Issue by PetrDlouhy** [django-nose/django-nose#254]
The following exception is thrown on Django 1.10:
```
  File "/home/travis/build/django-nose/django-nose/django_nose/runner.py", line 143, in _get_options
    django_opts = [opt.dest for opt in BaseCommand.option_list] + ['version']
AttributeError: type object 'BaseCommand' has no attribute 'option_list'
```

**Comment by jleclanche** [comment]
@jwhitlock Could you do a new release? Django 1.10a1 is available now, and this functionality isnt available in pypi.

**Comment by jwhitlock** [comment]
Released [django-nose 1.4.4](https://pypi.python.org/pypi/django-nose/1.4.4), which adds support for the [Django 1.10](https://docs.djangoproject.com/en/dev/releases/1.10/) beta.
